sdk/src/types/block/output/nft.rs

```diff
@@ -424,6 +420,17 @@ impl NftOutput {
     }
 }
 
+impl StorageScore for NftOutput {
+    fn storage_score(&self, params: StorageScoreParameters) -> u64 {
+        params.output_offset()
+            + 1 // Type byte
```

Comment:
Thinking out loud here, do you think the storage score of outputs should be without the 1 and we also have storage score with the 1 for the enum? It feels a bit weird to consider the type byte here when we usually don't